casts/splitNavigator/src/SplitNavigator/ScreenFallback.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { StyleProp, View, ViewProps, ViewStyle } from 'react-native';
+import { ResourceSavingView } from '@react-navigation/elements';
+
+type Props = {
+    visible: boolean;
+    children: React.ReactNode;
+    enabled: boolean;
+    style?: StyleProp<ViewStyle>;
+};
+
+let Screens: typeof import('react-native-screens') | undefined;
+
+try {
+    Screens = require('react-native-screens');
+} catch (e) {
+    // Ignore
+}
+
+export const MaybeScreenContainer = ({
+    enabled,
+    ...rest
+}: ViewProps & {
+    enabled: boolean;
+    children: React.ReactNode;
+}) => {
+    if (Screens?.screensEnabled?.()) {
+        return <Screens.ScreenContainer enabled={enabled} {...rest} />;
+    }
+
+    return <View {...rest} />;
+};
+
+export function MaybeScreen({ visible, children, ...rest }: Props) {
+    if (Screens?.screensEnabled?.()) {
+        return (
+            <Screens.Screen activityState={visible ? 2 : 0} {...rest}>
```

Comment:
Cool, I didn't know about the `activityState`: https://github.com/software-mansion/react-native-screens/blob/main/guides/GUIDE_FOR_LIBRARY_AUTHORS.md